### ISSUE TITLE
Group visibility setting is not honored

### ISSUE THREAD

**Issue by chairstacker** [estevez-dev/ha_client_pub#28]
I have an automation that prevents me from accidentally opening my garage door by changing the visibility of the group the cover is in.
The app always shows the group (cover), independently of the setting of the visibility setting.

Example code is:
```
- alias: Visibility Garage Door Switch Off
  trigger:
    platform: state
    entity_id: input_boolean.garage_door_cover_visible
    to: 'off'
  action:
    service: group.set_visibility
    entity_id: group.garage_door
    data:
      visible: False
```

**Comment by estevez-dev** [comment]
Fixed in 0.3.3
Please test and confirm after update

**Comment by chairstacker** [comment]
Confirmed working.
Thanks!
